src/uiUpdater.ts

```diff
@@ -196,9 +216,13 @@ export function loopTasks(selectedProject: Project) {
 
         inputValue?.addEventListener('keypress', function(event){
             if(event.key === 'Enter'){
-                if(inputValue?.value)
+                if(inputValue?.value && inputValue.value.trim() !== ''){
                 task.description = inputValue?.value
                 loopTasks(project)
+                }else{
+                    task.description = task.description
```

Comment:
what does this do?

Reply:
#also fixed a bug where you can enter a space for the task description and title and lose the ability to change them.